students/guillaume/quarter_2_project/leankitbackend/leankitbackend/util.py

```diff
@@ -0,0 +1,290 @@
+import json
+import re
+import csv
+import xlsxwriter
+from os import remove
+from datetime import datetime
+
+'''
+This module provides helper functions
+Some of these functions are being used for
+test purpose
+'''
+
+
+def printd(dic):
+    '''
+    Print a dictionnary in a more readable way
+    '''
+    for key, val in dic.items():
+        print('{} {}'.format(key, val))
+
+
+def printl(lst):
```

Comment:
check out the prettyprint module

```
from pprint import pprint

pprint(something)
```
saves you having to write this yourself...